cmd/nri-prometheus/versioninfo.json

```diff
@@ -1,44 +0,0 @@
-{
-	"FixedFileInfo":
-	{
```

Comment:
Why removing this? Asking just in case

Reply:
Not sure exactly what this file is, but it was accidentally included in the [commit](https://github.com/newrelic/nri-prometheus/pull/103) , and i checked that the pipeline remove it in the [clean step](https://github.com/newrelic/nri-prometheus/blob/9ace92f5754afe4dfc5c0e82d2385d5ac8476475/build/release.mk#L18)